qt-models/maplocationmodel.cpp

```diff
@@ -8,6 +8,9 @@ const char *MapLocation::PROPERTY_NAME_NAME       = "name";
 
 MapLocation::MapLocation()
 {
+	m_uuid = 0;
+	m_name = "";
```

Comment:
This one is not correct. QString has a default constructor which initializes to the empty string. No point in assigning an empty C-string to it.

Reply:
you are right. fixing it.